.husky/post-commit

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+# Generate contract TypeScript definition
```

Comment:
Is there a need to leave in these comments?

Reply:
I can remove them if you don't like.

Reply:
Yes, any commented code must be removed before merging, or the reason for leaving it in must be documented.